lib/ReactTags.js

```diff
@@ -56,6 +56,42 @@ class ReactTags extends React.Component {
     this.setState({ query })
   }
 
+  handlePaste (e) {
+    // allow over-ride, if there is a need
+    if (this.props.onPaste) {
+      return this.props.onPaste(e)
```

Comment:
That had been by understanding too. Maybe the way forward would be to go for consistency as you suggested and maybe replace and deprecate as a block in a future version?